watertrading/code/RunCSVconversion.py

```diff
@@ -0,0 +1,8 @@
+import json
+import pandas as pd
+import Utilities
+
+input_dir = "C:\\Users\\Philip\\Documents\\ProjectPARETO\\WSBidding_cases\\NE_PA_04"
```

Comment:
Will other users ever be expected to run this code? If so, I don't figure it would work for them, with this path from your machine being hardcoded.

Reply:
No, this code doesn't need to be part of the repo; it's just a script calling a function to let me process data quickly. Users would ostensibly use the function in the Utilities themselves.